rllib/execution/train_ops.py

```diff
@@ -61,9 +62,11 @@ def __call__(self,
             if self.num_sgd_iter > 1 or self.sgd_minibatch_size > 0:
                 lw = self.workers.local_worker()
                 info = do_minibatch_sgd(
-                    batch, {pid: lw.get_policy(pid)
-                            for pid in self.policies}, lw, self.num_sgd_iter,
-                    self.sgd_minibatch_size, [])
+                    batch, {
+                        pid: lw.get_policy(pid)
```

Comment:
Did you try this for PPO? Don't think this line of code returns the right policies as intended.